tests/CDN/PantherPortalTest.php

```diff
@@ -50,10 +56,12 @@ public function testException(): void
     }
 }
 
-class ClientSpy extends \SoapClient
```

Comment:
I don't understand why we have a class, if we mock it anyway, can't we directly mock it or better create an anonymous class:
```php
        $client = new class() implements \SoapClient {
            public function flush(): string
            {
                return 'hello';
            }
        };
```

WDYT @VincentLanglet @franmomu ?